plugin/audit_null/audit_null.cc

```diff
@@ -130,6 +131,11 @@ static char *g_record_buffer;
 
 #undef AUDIT_NULL_VAR
 
+static const constexpr size_t event_response_buffer_len = 1000;
```

Comment:
Is 1000 enough now when we have certificates also included in GENERAL STATUS event?

Reply:
For now yes, as all other fields are short.